### ISSUE TITLE
"keeps stopping" in keepass DX

### ISSUE THREAD

**Issue by sleroq** [zhanghai/MaterialFiles#720]
Can't open KeePass DX If Material Files is not running

I get this error
![image](https://user-images.githubusercontent.com/52239427/159696092-8072dbab-ed14-4135-9225-0d55547e0f05.png)

But if I open Material Files before starting keepass DX, then everything works

---

Android 10 LineageOs 17.1

**Comment by zhanghai** [comment]
Thanks for reporting this. This is a duplicate of #716 and will be fixed in the next release.
